### PR TITLE
Email auth updated translations.

### PR DESCRIPTION
**Pull request by ice-myles**
Email auth updated translations.

### CODE REVIEW

File: application.yaml [hunk 1]
```diff
@@ -93,6 +93,9 @@ auth/email-link:
     jwtSecret: bogus
   confirmationCode:
     maxWrongAttemptsCount: 3
+  PetName: Tenanter
+  AppName: Tenant
+  TeamName: Tenant
```
Comment (ice-ares) [inline application.yaml]: U sure those are the correct keys?

Comment (ice-myles) [reply application.yaml]: Do you mean Tenanter, Tenant, Tenant? I expected that they will be different for each tenant, so to call them like 1 of the tenants not right, and better to name them commonly. No?

Comment (ice-ares) [reply application.yaml]: they're uppercased keys, not sure it will bind to the cfg struct

Comment (ice-myles) [reply application.yaml]: Oh, right. It works, as I've tested, but it should be fixed ofc.

**Pull request merged**